content/blog/2020-04-usethis-1-6-0/index.Rmarkdown

```diff
@@ -0,0 +1,96 @@
+---
+title: usethis 1.6.0
+author: Hadley Wickham
+date: '2020-04-06'
+slug: usethis-1-6-0
+categories:
+  - package
+tags:
+  - devtools
+  - usethis
+photo:
+  author: Rock'n Roll Monkey
+  url: https://unsplash.com/photos/R4WCbazrD1g
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We're happy to announce that [usethis](https://usethis.r-lib.org) 1.6.0 is now available on CRAN. usethis is a package that facilitates interactive workflows for R project creation and development. It's mostly focussed on easing day-to-day package development, but many of its functions are also useful for non-package projects. Install usethis with:
+
+```{r eval = FALSE}
+install.packages("usethis")
+```
+
+This blog post discusses three sets of improvements to usethis:
+
+* Our helpers for using GitHub Actions are now polished, and if you're an 
+  R package developer who uses GitHub, we recommend moving away from Travis in 
+  favour of GitHub Actions.
+  
+* We've made a number of small tweaks to `create_package()` in order to reduce
+  inconsistencies during the initial startup phase of a package.
+  
+* We've continued to polish our tools for contributing to other people's
+  packages.
+  
+This release also includes a handful of new functions (my favourite is [`rename_file()`](https://usethis.r-lib.org/reference/rename_files.html)), many bug fixes, and lots of small improvements. We're slowly grinding down the rough edges, so usethis just works in more cases, and when it fails, it's more likely to give you error messages that help you quickly figure out the problem. As usual, you can find detailed notes about all these changes in the [change log](https://usethis.r-lib.org/news/index.html).
+
+## GitHub Actions
+
+[GitHub Actions](https://github.com/features/actions) allows you to automatically run code whenever you push to GitHub. If you're developing a package this allows you to automate tasks like:
+
+* Run `R CMD check` on multiple platforms (Linux, Windows, and Mac).
+* Record the code coverage of your unit tests.
+* Re-build your [pkgdown](https://pkgdown.r-lib.org/) website.
+
+Outside of a package, you can also use them to rebuild blogdown and bookdown sites, or regularly re-knit `.Rmd` files. 
+
```

Comment:
I think this is mostly good as is, maybe the only thing we should consider adding is something comparing GHA to travis, e.g.

Compared to Travis-CI GitHub Actions provides more resources. 6 hour build times and 20 concurrent builds on GitHub Actions vs 50 minute build times and ~5 concurrent builds on Travis-CI. GitHub Actions also has more complete support for building on Windows and macOS and a more natural way of using Docker containers. Also because the workflows are run by GitHub extra setup and authentication is unnecessary.